darwin/device.go

```diff
@@ -516,7 +517,7 @@ func (d *Device) HandleXpcEvent(event xpc.Dict, err error) {
 				sub.fn(args.data())
 			}
 			break
-		}
+		// }
 		c.rspc <- m
```

Comment:
In other words, this line can, and should be removed, as it is no longer needed.

Reply:
@moogle19 done, but then that break means the go channel thing on the next line is never happening..

Reply:
The line `c.rspc <- m` is to send message m to the channel c.rspc if not handled by the preceding code. Since your change handled m, no need to send now. That is why go get says that line should be removed.